airbrake/utils.py

```diff
@@ -16,6 +17,15 @@
     TypeType = type
 
 
+class FailProofJSONEncoder(json.JSONEncoder):
+    """Uses object's representation for unsupported types."""
+
+    # pylint: disable=E0202
```

Comment:
I disabled E0202 for this block instead of workarounding it, to comply with the python documentation (for overriding JSONEncoder): https://docs.python.org/2/library/json.html#json.JSONEncoder.default